Frontend/src/app/components/broker/v2-panel/dual-pane-chart/dual-pane-chart.component.scss

```
@@ -307,6 +309,32 @@
+  padding: 0.38rem 0.55rem;
+  border: 1px solid color-mix(in srgb, var(--border-light) 90%, transparent);
+  border-radius: var(--radius);
+  background: color-mix(in srgb, var(--bg-surface) 88%, transparent);
+  color: var(--text-secondary);
+  font: 600 var(--fs-xxs)/1.2 var(--font-mono);
+  list-style: none;
+  pointer-events: none;
+  backdrop-filter: blur(5px);
+
+  li { display: inline-flex; align-items: center; gap: 0.32rem; white-space: nowrap; }
+  i { width: 0.5rem; height: 0.5rem; border-radius: 50%; box-shadow: 0 0 7px currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)